.github/workflows/update_ci_image.yaml

```diff
@@ -6,13 +6,13 @@ on:
     # 7am UTC, 12am PDT
     - cron:  '0 7 * * *'
   push:
-    branches:
-      - main
-    paths:
-      - '**/package.xml'
-      - '**/*.repos'
-      - 'Dockerfile'
-      - '.github/workflows/update_ci_image.yaml'
+    # branches:
```

Comment:
Readd?

Reply:
Commented out temporarily while WIP to avoid wasting credits. Will revisit before merging.